components/WeeklyPickButton/WeeklyPickButton.test.tsx

```diff
@@ -3,19 +3,34 @@ import { render, screen } from '@testing-library/react';
 import { WeeklyPickButton } from './WeeklyPickButton';
 import { RadioGroup } from '../RadioGroup/RadioGroup';
 
+const weeklyPickData = {
+  team: 'Ravens',
+  src: '/path/to/ravens.svg',
+  height: '48',
+  width: '48',
+  alt: 'Ravens',
+  className:
+    'text-base font-normal leading-none text-zinc-50 cursor-pointer flex gap-2 items-center rounded-xl border-2 border-zinc-800 peer-aria-checked:border-orange-600 py-4 px-3 w-full peer-hover:bg-zinc-800 transition',
+};
+
 describe('WeeklyPickButton', () => {
   it('renders correctly', () => {
     render(
       <RadioGroup>
```

Comment:
m: Still need to remove RadioGroup. This has nothing to do with the test for this component.

Reply:
RadioGroup is required by shadcn